src/core.ts

```diff
@@ -101,8 +101,6 @@ export const render = (component: any, parent: HTMLElement | null = null, remove
         })
       else appendChildren(parent, _render(el))
     }
-    //@ts-ignore
-    if (parent.ssr) return parent.ssr
```

Comment:
If the `render` and `h` functions in SSR take rendered HTML as the return value, it is impossible to determine whether the parent component should escape or not when rendering its children. Therefore, the HTMLElemetSSR is used as the return value. Since HTMLElementSSRs are eventually stringified, this change does not change the API in any way.

What do you think?